llvm/tools/llvm-cxxfilt/llvm-cxxfilt.cpp

```diff
@@ -20,6 +20,7 @@
 #include "llvm/TargetParser/Triple.h"
 #include <cstdlib>
 #include <iostream>
+#include <string>
```

Comment:
Nit: `std::string` is already used by this file, so adding this seems unnecessary by my reading of the include rules in the style guide.

Reply:
This is added automatically by `clangd` LSP server using the project configuration, so I guess it is cleaner according to the official coding style but I can remove it.